src/components/IntentRedirect.jsx

```diff
@@ -18,17 +18,25 @@ const IntentRedirect = ({ installedKonnectors, location }) => {
         return accumulator
       }, {})
 
-  if (query.konnector) {
-    if (
-      installedKonnectors.find(konnector => konnector.slug === query.konnector)
-    ) {
-      return <Redirect to={`/connected/${query.konnector}`} />
-    } else {
-      cozy.client.intents.redirect('io.cozy.apps', { slug: query.konnector })
-    }
+  if (!query.konnector) {
+    return <Redirect to={`/connected`} />
   }
 
-  return <Redirect to={`/connected`} />
+  if (
+    !installedKonnectors.find(konnector => konnector.slug === query.konnector)
```

Comment:
Maybe check here if query.konnector is defined?

Reply:
That's why the early return above is for :)